src/components/Listing/Listing.html

```diff
@@ -0,0 +1,38 @@
+<div class="container">
+  <div class="advertiser-details">
+    <h1 class="title">{{ listing["listing"]["title"] }}</h1>
```

Comment:
Just a doubt: Can you access the attributes with dot `.`?
If you can, then consider this way, because it is common to find.

Like this:

```suggestion
    <h1 class="title">{{ listing.listing.title }}</h1>
```

Consider this for the all HTML files, please.

Reply:
Yeah, I don't know why yet, but it does not work with dot :/